apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -465,23 +464,24 @@ await spannerTransaction.ExecuteMutationsAsync(mutations, cancellationToken, com
 
         private List<Mutation> GetMutations()
         {
+            // If SpannerConversionOptions is involved, build it from the connection.
+            // SpannerConversionOptions options = SpannerConversionOptions.ForConnection(SpannerConnection);
+            SpannerConversionOptions conversionOptions = null;
```

Comment:
Not sure I understand these lines? Is it meant to be a TODO?

Reply:
Not quite - it's effectively a hint if we ever change ToProtobufValue. It's not *currently-outstanding* work which TODO would imply. I'll clarify the comment.